### PR TITLE
revisions based on review start 

### PR DESCRIPTION
**Pull request by jzf2101**
- added citations for reproducibility crisis
- took initial wording suggestions
- added institutions

### CODE REVIEW

**Comment by jzf2101** [comment]
@andrewosh what institution should we use for you?

**Comment by andrewosh** [comment]
@jzf2101 If it's acceptable, "Independent" would be the most accurate :) My work on Binder was done at the Janelia Research Campus, though I'm not currently affiliated.

Also, congrats on the awesome paper/work!

**Comment by jzf2101** [comment]
@freeman-lab @mpacer @Carreau  @choldgraf @fperez @ellisonbg could you please confirm your institutions as well? I've tried to harmonize them for simplicity so for example all Berkeley people are with the same name since it's the same university.

File: papers/project_jupyter/paper.rst [hunk 1]
```diff
@@ -30,42 +30,42 @@
...
+:institution: UC Berkeley
 :equal-contributor:
 
 :author: Fernando Perez
 :email:
-:institution:
+:institution: UC Berkeley
 :equal-contributor:
 
 :author: Benjamin Regan Kelley
```
Comment (willingc) [inline papers/project_jupyter/paper.rst:61]: @minrk's name has some typos.

**Pull request merged**